react/Banner.js

```diff
@@ -7,57 +7,51 @@ import { Link } from 'render'
  */
 class Banner extends Component {
   render() {
-    const { image, page, description, targetParams } = this.props
+    const {
+      image,
+      page,
+      description,
+      targetParams,
+      mobileImage,
+      mobileHeight,
+    } = this.props
 
     return (
       <div>
         <Link page={page} params={targetParams}>
-          <img width="100%" src={image} alt={description}/>
+          <div className="img-container">
+            <img
+              width="100%"
```

Comment:
Have you tried using tachyons class `w-100`?

Reply:
👍

Reply:
👍